kavaplugin/src/main/java/com/kaltura/playkit/plugins/kava/KavaAnalyticsPlugin.java

```diff
@@ -294,11 +294,13 @@ private void handleStateChanged(PlayerEvent.StateChanged event) {
                 //We should start count buffering time only after IMPRESSION was sent.
                 if (isImpressionSent) {
                     dataHandler.handleBufferingStart();
+                    sendAnalyticsEvent(KavaEvents.BUFFER_START);
                 }
                 break;
             case READY:
                 playerState = PlayerState.READY;
                 dataHandler.handleBufferingEnd();
+                sendAnalyticsEvent(KavaEvents.BUFFER_END);
```

Comment:
@GouravSna 
please keep flag one buffer start
so only if on you will fire buffer end 
andthe set it to off
so we will not fire end buffer each time that player is ready

Reply:
Fixed